java/lib/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    // Apply the kotlin-jvm plugin for Kotlin JVM projects.
+    kotlin("jvm") version "1.6.21"
+    // Apply the maven-publish plugin for publishing the library.
+    id("maven-publish")
```

Comment:
we might need these two also. Signing one might be necessary if we are going to publish on maven central
id("java-library")
id("signing")